tools/virtualization/offloading-domain-tty-output-to-console-with-custom-kernel-initramfs-images.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Offloading domain TTY output to console with custom kernel/initramfs images  
+subsection: virtualization  
+order: 12  
+---
+
+# Offloading domain TTY output to console with custom kernel/initramfs images
```

Comment:
I think this would deserve more thorough description of what this sub-page aims to do.

Reply:
Addressed here https://github.com/developer-portal/content/pull/394/commits/35e9087c5bbc75e08e388490e93378c5b2300783